amethyst_renderer/src/renderer.rs

```diff
@@ -145,6 +155,22 @@ impl Renderer {
         &mut self.events
     }
 
+    /// Resize the targets associated with this renderer and pipeline.
+    pub fn resize<P: PolyPipeline>(&mut self, pipe: &mut P, new_size: (u32, u32)) {
+        self.main_target.resize_main_target(&self.window);
+        let mut targets = HashMap::default();
+        targets.insert("".to_string(), self.main_target.clone());
+        for key in pipe.targets().keys() {
+            if key != "" {
+                let (key, target) = TargetBuilder::new(key.clone())
+                    .build(&mut self.factory, new_size)
+                    .unwrap();
```

Comment:
Is there any way this could fail?

Reply:
If this was going to fail it would have failed when the game started and tried to create the target rather than later at the point of resizing.

Reply:
If you're sure about that, let's leave `unwrap` here then.